<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chemistry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>1052</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Эксперименты</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Таблица экспериментов</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="update_button_experiments">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Обновить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Первое вещество</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="firstElementEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="swapButton">
            <property name="text">
             <string>&lt;--&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Второе вещество</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="secondElementEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="filterButton">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QTableWidget" name="experimentsTab">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="textElideMode">
           <enum>Qt::ElideLeft</enum>
          </property>
          <property name="rowCount">
           <number>1</number>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>300</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>70</number>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>70</number>
          </attribute>
          <row/>
          <column>
           <property name="text">
            <string>  id  </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>   Первый элемент   </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>   Второй элемент   </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Тип процесса</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>   Температура, К   </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>   Давление, P   </string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>   Статья   </string>
           </property>
          </column>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="add_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Добавить данные об эксперименте</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Статьи</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>70</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>26</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Информация о статьях</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>поиск</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Добавить статью</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QListWidget" name="articles_list">
          <property name="minimumSize">
           <size>
            <width>500</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>2000</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Таблица веществ</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1" rowspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Таблица веществ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="update_button_elements">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Обновить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>поиск</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>найти</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add_element_button">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string>Добавить</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableWidget" name="elementsTab">
            <property name="minimumSize">
             <size>
              <width>1050</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1050</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="rowCount">
             <number>2</number>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>300</number>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>500</number>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="verticalHeaderMinimumSectionSize">
             <number>150</number>
            </attribute>
            <row/>
            <row/>
            <column>
             <property name="text">
              <string>Имя</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Свойства</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>162</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>163</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Попытки расчетов</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>220</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="attempt_label_1">
               <property name="text">
                <string>Модель</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="modelComboBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="attempt_label_2">
               <property name="text">
                <string>Метод оптимизации</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="methodsComboBox">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="attempt_label_3">
                 <property name="text">
                  <string>№ Эксп.</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="id_exp_edit">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="attempt_label_4">
             <property name="text">
              <string>Начальные данные:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLineEdit" name="init_edit">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>500</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>&quot;a12&quot;:0,&quot;a21&quot;:0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="multistartCheckBox">
               <property name="text">
                <string>Мультизапуск</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="attempt_label_5">
                 <property name="text">
                  <string>Мин. значения:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="min_edit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>1000</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&quot;a12&quot;:0,&quot;a21&quot;:0</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="attempt_label_6">
                 <property name="text">
                  <string>Макс. значения:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="max_edit">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>1000</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>&quot;a12&quot;:1000,&quot;a21&quot;:1000</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="attempt_label_7">
                 <property name="text">
                  <string>Количество</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="count_edit">
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>16777215</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>173</width>
               <height>131</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QPushButton" name="calculateButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string>Рассчитать</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTabWidget" name="attemptsTabWidget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
          <property name="movable">
           <bool>true</bool>
          </property>
          <property name="tabBarAutoHide">
           <bool>false</bool>
          </property>
          <widget class="QWidget" name="tab_6">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="tab_7">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
